pythonx/jedi_vim.py

```diff
@@ -724,9 +724,17 @@ def show_documentation():
     if not definitions:
         echo_highlight('No documentation found for that.')
         vim.command('return')
-    else:
-        docs = ['Docstring for %s\n%s\n%s' % (d.desc_with_module, '=' * 40, d.docstring())
-                if d.docstring() else '|No Docstring for %s|' % d for d in definitions]
+        return
```

Comment:
This is probably too much and should be removed :) (Probably the whole if/else)

Reply:
@davidhalter 
What do you mean exactly?
I'd rather not change it too much - I wanted to investigate only why Jedi master breaks it.

And it should be rewritten to use `script.help()` instead then better anyway.

Reply:
Oh, I realized that this is probably ok, I didn't really understand what was changed here.